.github/workflows/build.yaml

```diff
@@ -181,12 +188,17 @@ jobs:
           log-tarball-prefix: ${{matrix.cfg.name}}
           tests-label: nonparallelizable_tests
           test-timeout: 420
+      - name: Export core dumps
+        run: docker run --mount type=bind,source=/var/lib/systemd/coredump,target=/cores alpine sh -c 'tar -C /cores/ -c .' | tar x
```

Comment:
I know this looks weird, and to add more confusion it's different than the parallel tests job because NP/LR tests don't run the job in a container until `parallel-ctest-containers` step.

On ENF runners the user is _not_ `root` but rather a user named `enf` that does not have sudo privileges (this is unlike GitHub runners). But `/var/lib/systemd/coredump` files are only readable by `root`. Since docker is not creating a userns in this case, we can use docker as a cheat to get access to the files without `sudo` and without addtional modificatons on the ENF runners.